### DISCUSSION TITLE
failed to bind to address

### DISCUSSION THREAD

**Discussion by senorsmile** [any1/wayvnc#169] [Q&A] [answered]
I'm running sway on Nixos 22.05.  

I created the certs and a config file as described.  
When I start wayvnc I get the error

```
$ wayvnc -C ./config
ERROR: Failed to bind to address
```

Even if I change the bind address 127.0.0.1 I get the same error.

**Comment by any1** [comment]
What does `netstat -ntl` or `ss -ntl` look like on your system?

**Comment by senorsmile** [comment, marked as answer]
Ah, it shows that qemu is bound to port 5900.  Weird since it's not actively running at the moment.  
```
sudo ss -tlnp | grep 5900
LISTEN 0      1                                                 127.0.0.1:5900       0.0.0.0:*    users:((".qemu-system-x8",pid=370922,fd=21)) 
```

How do I bind it to a different port? 

I tried `address=127.0.0.1:5901` to no avail.

**Reply by any1** [reply]
`port=5901`

**Reply by senorsmile** [reply]
Thanks!  That was definitely the param needed.  Now I need to forget out auth for the client, but turning off auth altogether gets me in.  Thanks!
